build/templates/_grpc_stub_interpreter.py/initialization_method.py.mako

```diff
@@ -0,0 +1,2 @@
+<%page args="f, config, method_template"/>\
+<%include file="/_grpc_stub_interpreter.py/default_method.py.mako" args="f=f, config=config, method_template=method_template" />\
```

Comment:
TODO: revert this file (and the next) before merging in Dan's (still pending) changes

Reply:
Done!